include/libsemigroups/containers.hpp

```diff
@@ -30,6 +30,88 @@
 
 namespace libsemigroups {
   namespace detail {
+    template <typename T, size_t N>
+    class StaticTriArray2 {
```

Comment:
```suggestion
    class StaticTriVector2 {
```
Change name to better reflect that the class implements a `Vector2` interface.

Reply:
Thanks for pointing this out! This name change has been made.